### DISCUSSION TITLE
How to parse entire JSON body?

### DISCUSSION THREAD

**Discussion by Just-Insane** [adnanh/webhook#556] [Q&A]
I'm trying to take a Grafana webhook, and parse the entire body, which should all be JSON.

Unfortunately, I haven't been able to find any documentation on what the generic "webhook" alert JSON body looks like, as this would be much easier.

**Comment by go2tom42** [comment]
You can use this site to see what the payload looks like  
https://webhook.site

Or you can dumb the raw payload to a file from adnanh/webhook

Needed in hooks.json
```
    "id": "test",
    "execute-command": "/scripts/test.sh",
    "command-working-directory": "/scripts",
    "pass-file-to-command": 
    [
      {
        "source": "raw-request-body",
        "envname": "PFILE"
      }
    ]
```

needed in test.sh
```
#!/bin/sh

cat "$PFILE"  >> "./test.json"
```
